blocks/library/latest-comments/style.scss

```diff
@@ -0,0 +1,23 @@
+
+.editor-visual-editor__block[data-type="core/latest-comments"] {
+
+	.wp-block-latest-comments {
+		padding-left: 2.5em;
+	}
+
+	.wp-block-latest-comments__comment-avatar {
+		display: block;
+		float: left;
+		height: 48px;
+		width: 48px;
+		border-radius: 24px;
+		margin-right: 5px;
```

Comment:
Is it fine that these sizes are in px? If not, can you give me the requested values as em?